packages/migrate/package.json

```diff
@@ -57,12 +54,16 @@
     "@prisma/debug": "workspace:*",
     "@prisma/get-platform": "3.8.0-43.34df67547cf5598f5a6cd3eb45f14ee70c3fb86f",
     "@sindresorhus/slugify": "1.1.2",
+    "chalk": "4.1.2",
     "execa": "5.1.1",
     "has-yarn": "2.1.0",
     "indent-string": "4.0.0",
     "log-update": "4.0.0",
+    "mariadb": "2.5.5",
+    "mssql": "7.3.0",
     "new-github-issue-url": "0.2.1",
     "open": "7.4.2",
+    "pg": "8.7.1",
```

Comment:
Are these really used outside of tests?

Reply:
Same as https://github.com/prisma/prisma/pull/11156#discussion_r784860441